### DISCUSSION TITLE
Beispiel Projekt

### DISCUSSION THREAD

**Discussion by ChrisKoh83** [LeonardSchwenk/public-gruppenadressen-generator#33] [Ideas]
Wäre es denkbar für Einsteiger in das Thema ein Beispiel anzeigen zu können?

Daran könnte man lernen sein eigenes Projekt zu erstellen.

**Comment by LeonardSchwenk** [comment]
Gefällt mir sehr, der Gedanke. Mir ist auch schon in den Sinn gekommen, es zu ermöglichen, Projekte und damit dann natürlich auch Beispiele zu teilen. Um sich einfacher austauschen zu können und den Einstieg zu erleichtern.

**Comment by LeonardSchwenk** [comment]
@ChrisKoh83 
Ist schon länger eingebaut :), bestehende Nutzer können anfordern ein Beispielprojekt zu bekommen und für neue Registrierungen wird automatisch eins erstellt :).
